src/assets/images/icons/web/warning.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
When comparing the fonts it seems that we are using the wrong font for the messages